### PR TITLE
UI update

### PR DESCRIPTION
**Pull request by tamuda**

### CODE REVIEW

File: main31.js [hunk 1]
```diff
@@ -1,73 +1,84 @@
...
+  style,
+  setColorsFromThemeName,
+} from "./assets15.js";
 
 let loadInterval;
 var chatbotID = "";
-//const BASE_URL = "http://localhost:5001/dev/";
-const BASE_URL = "https://sswj8m0la3.execute-api.af-south-1.amazonaws.com/dev/";
+const BASE_URL = "http://localhost:5001/dev/";
+//const BASE_URL = "https://sswj8m0la3.execute-api.af-south-1.amazonaws.com/dev/";
```
Comment (samzi123) [inline main31.js]: Please switch which base url is being used, as we want this to be the live version

File: main31.js [hunk 2]
```diff
@@ -1,73 +1,84 @@
...
+      try {
+        const bodyJson = JSON.parse(body);
+        console.log("chatbot info: ", bodyJson);
+        chatbotInfo = bodyJson;
+        document.getElementsByClassName("penpal-header")[0].innerHTML =
+          chatbotInfo.chatTitle;
+        document.getElementsByClassName("penpal-first-message")[0].innerHTML =
+          chatbotInfo.welcomeMessage;
+        //setColorsFromThemeName(chatbotInfo.colorScheme);
+        setColorsFromThemeName("dark");
```
Comment (samzi123) [inline main31.js]: The line above should not be commented out, as we want to set the theme to the one fetched from the DB, rather than being hardcoded.

**Review by samzi123 — CHANGES_REQUESTED** [review]
Overall great work, just a couple of comments. Also, please take a look at how the input textbox scales on mobile screens, as it is very small. See the attached image. Consider adding a breakpoint so that it is bigger on mobile devices.


Screenshot 2023-08-01 at 00 39 38

**Comment by tamuda** [comment]
I have made the widget responsive

**Review by samzi123 — APPROVED** [review]

**Pull request merged**